collector/lib/SysdigService.cpp

```diff
@@ -133,7 +133,10 @@ sinsp_evt* SysdigService::GetNext() {
   if (event->get_category() & EC_INTERNAL) return nullptr;
 
   // TODO (rc) not needed when syscall filtering implemented in the eBPF probe
-  if (useEbpf && !global_event_filter_[event->get_type()]) return nullptr;
+  // if (useEbpf && !global_event_filter_[event->get_type()]) {
```

Comment:
note: verify that this truly can be removed, and remove before merging.

Reply:
looks like this is still required because of RHEL 7 still using sys_enter and sys_exit. However, it doesn't seem to have an effect on other platforms. I'll update the comment to reflect this situation.

Reply:
Need to work out a good way of doing this, because it's still relevant to RHEL 7 , which is using sys_enter/sys_exit still.

Reply:
Maybe we can simply add an `isRHEL7` variable somewhere, set it at startup with a new heuristic and check it here alongside `useEbpf`? There might be an even easier solution, but it eludes me right now.

Reply:
I'm sure there's actually an isRHEL7 method in the HostInfo class so I can probably just use that to enable this filter.